docs/src/app/components/pages/components/BottomSheet/ExampleCloseable.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import BottomSheet from 'material-ui/BottomSheet';
+import MenuItem from 'material-ui/MenuItem';
+import RaisedButton from 'material-ui/RaisedButton';
+
+export default class BottomSheetCloseableExample extends React.Component {
```

Comment:
We use the following convention in our other examples: `BottomSheetExampleCloseable`.